lib/usbd/usbd.c

```diff
@@ -224,6 +234,8 @@ bool usbd_is_vbus(usbd_device *dev){
 }
 
 void usbd_enable(usbd_device *dev, bool onoff){
+	if (!onoff)
+		usbd_disconnect(dev, false);
 	if (dev->backend->power_control)
 		dev->backend->power_control(dev, (onoff)? usbd_paActivate : usbd_paShutdown );
 }
```

Comment:
yeah, see alredy. more trouble now bother me - cant`t understand how to wake up FS from powerdown, cause it wan`t normaly detect online and reset after ungating core clock